### ISSUE TITLE
[fail] MySQLSlickOffsetStoreSpec

### ISSUE THREAD

**Issue by ignasi35** [akka/akka-projection#320]
Happened on a PR build

```
[info] akka.projection.slick.MySQLSlickOffsetStoreSpec *** ABORTED *** (6 seconds, 413 milliseconds)
[info]   java.util.concurrent.TimeoutException: Future timed out after [3 seconds]
[info]   at scala.concurrent.impl.Promise$DefaultPromise.tryAwait0(Promise.scala:212)
[info]   at scala.concurrent.impl.Promise$DefaultPromise.result(Promise.scala:225)
[info]   at scala.concurrent.Await$.$anonfun$result$1(package.scala:201)
[info]   at scala.concurrent.BlockContext$DefaultBlockContext$.blockOn(BlockContext.scala:62)
[info]   at scala.concurrent.Await$.result(package.scala:124)
[info]   at akka.projection.slick.SlickOffsetStoreSpec.beforeAll(SlickOffsetStoreSpec.scala:195)
[info]   at org.scalatest.BeforeAndAfterAll.liftedTree1$1(BeforeAndAfterAll.scala:212)
[info]   at org.scalatest.BeforeAndAfterAll.run(BeforeAndAfterAll.scala:210)
[info]   at org.scalatest.BeforeAndAfterAll.run$(BeforeAndAfterAll.scala:208)
[info]   at akka.projection.slick.SlickOffsetStoreSpec.org$scalatest$wordspec$AnyWordSpecLike$$super$run(SlickOffsetStoreSpec.scala:172)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.$anonfun$run$1(AnyWordSpecLike.scala:1190)
[info]   at org.scalatest.SuperEngine.runImpl(Engine.scala:535)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.run(AnyWordSpecLike.scala:1190)
[info]   at org.scalatest.wordspec.AnyWordSpecLike.run$(AnyWordSpecLike.scala:1188)
[info]   at akka.projection.slick.SlickOffsetStoreSpec.run(SlickOffsetStoreSpec.scala:172)
[info]   at org.scalatest.tools.Framework.org$scalatest$tools$Framework$$runSuite(Framework.scala:318)
[info]   at org.scalatest.tools.Framework$ScalaTestTask.execute(Framework.scala:513)
[info]   at sbt.TestRunner.runTest$1(TestFramework.scala:139)
[info]   at sbt.TestRunner.run(TestFramework.scala:154)
[info]   at sbt.TestFramework$$anon$3$$anonfun$$lessinit$greater$1.$anonfun$apply$1(TestFramework.scala:317)
[info]   at sbt.TestFramework$.sbt$TestFramework$$withContextLoader(TestFramework.scala:277)
[info]   at sbt.TestFramework$$anon$3$$anonfun$$lessinit$greater$1.apply(TestFramework.scala:317)
[info]   at sbt.TestFramework$$anon$3$$anonfun$$lessinit$greater$1.apply(TestFramework.scala:317)
```

**Comment by ignasi35** [comment]
After rerunning the job on the failing PR we merged the PR and the build for the merge commit in `master`failed again:

https://travis-ci.com/github/akka/akka-projection/jobs/358713406#L547

**Comment by seglo** [comment]
This was addressed in #380
